server.js

```diff
@@ -41,7 +65,38 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({extended: true}));
 app.use(bodyParser.raw());
 
+
+//setup passport
+
+passport.use(new OAuth2Strategy({
```

Comment:
prefer these (everything related to passport)  to go into auth/passportstrategies.js 

server.js should not accumulate clutter